src/asciidoc/docs/about/contribute.adoc

```diff
@@ -0,0 +1,176 @@
+
+We also have are a few extensions on AsciiDoc syntax added in to make the developer site ever better.
+
+=== Page Attributes
+Page attributes are used to add extra data related to a document.
+Add these to the top of your document, just under the document title.
+
+Page slug::
+	This corresponds to the page url and is used for linking between different pages. +
+	`:page-slug: example-page`
+Page description::
+	This is a short description of the content of the page and is used when sharing the link on social media and chat platforms. +
+	`:page-description: This page is an example`
+Page banner::
+	This references an image which is used as the thumbnail when sharing the link on social media and chat platforms. +
+	`:page-banner: \{imagesDir\}/example-page.png`
+
+NOTE: `\{imagesDir\}` refers to the image directory, which can be found in the repository at `static/images`
+
+TIP: If you have block images on your page you don't need to set the page banner, the first image will be used instead
+
+=== Tabs
+Sometimes it can be useful to sort your content into tabs.
+You'll see them used particularly often to display code snippets for differing frameworks.
+To add tabs into your document use the following syntax:
+
+[source, asciidoc]
+----
+[.tabs]
+--
+
+.Tab 1 title
+Tab 1 content.
+
+.Tab 2 title
+Tab 2 content.
+
+--
+----
+
+The above code will output the following:
+
+[.tabs]
+--
+
+.Tab 1 title
+Tab 1 content.
+
+.Tab 2 title
+Tab 2 content.
+
+--
+
+You will also notice that on some pages changing tabs will switch the tabs in multiple places (e.g. choosing a coding framework in a blueprint article). You can use scope attribute to sync multiple sets of tabs with one another, by setting two sets of tabs with the same scope attributes, whenever a tab is selected all the tabs with the same name will also be selected across matching scope names.
+
+[source, asciidoc]
+----
+[.tabs,scope="framework"]
+--
+
+.React
+Some react code
+
+.Vanilla JS
+Some Vanilla JS code
+
+--
+
+[.tabs,scope="framework"]
+--
+
+.React
+Some more react code. These tabs will sync with the above ones.
+
+.Vanilla JS
+Some more Vanilla JS code. These tabs will sync with the above ones.
+
+--
+----
+
+
+[.tabs,scope="framework"]
+--
+
+.React
+Some react code
+
+.Vanilla JS
+Some Vanilla JS code
+
+--
+
+[.tabs,scope="framework"]
+--
+
+.React
+Some more react code. These tabs will sync with the above ones.
+
+.Vanilla JS
+Some more Vanilla JS code. These tabs will sync with the above ones.
+
+--
+
+== Upgrading the site
+
+You can also contribute by making improvements to how the developer site run. The developer site is built using https://www.gatsbyjs.com/[Gatsby], https://asciidoctor.org/docs/asciidoctor.js/[Asciidoctor.js] and the https://www.gatsbyjs.com/plugins/gatsby-transformer-asciidoc/[gatsby asciidoc transformer].
+
+Download https://github.com/madewithkoji/docs[the GitHub repository] and start your development server by running `npm start`.
```

Comment:
I think we should add some instructions on how to preview the docs/site with Gatsby (gatsby develop and gatsby build / gatsby serve). We could also add a recommended step to the submission process for previewing the changes.